api-tests/src/steps/ipv-steps.ts

```diff
@@ -145,3 +147,21 @@ Then(
     }
   },
 );
+
+Then(
+  "my proven user details match",
+  async function (this: World): Promise<void> {
+    assert.ok(
+      isPageResponse(this.lastJourneyEngineResponse),
+      `got a ${describeResponse(this.lastJourneyEngineResponse)}`,
+    );
+    assert.equal("page-ipv-reuse", this.lastJourneyEngineResponse.page);
+
+    const provenIdentity = await getProvenIdentityDetails(this.ipvSessionId);
+
+    assert.ok(
+      doProvenDetailsMatchIdentity(provenIdentity, this.identity),
+      "Proven identity and current identity do not match.",
```

Comment:
You tend to get better error messages if you use `assert.equal` (or other more specific assertions) - it'll show you a diff of the two sides.

We could perhaps compute the 'expected' proven user identity (from `this.identity`) and pass them into `assert.deepStrictEqual`?

Should look something like
![image](https://github.com/user-attachments/assets/56a9680e-df0e-4a0c-9ef6-2286f4fcbdd0)